analyzers/its/expected/Ember-MM/EmberAPI-{208AA35E-C6AE-4D2D-A9DD-B6EFD19A4279}-S1067.json

```diff
@@ -67,27 +67,27 @@
 },
 {
 "id":  "S1067",
-"message":  "Reduce the number of conditional operators (4) used in the expression (maximum allowed 3).",
+"message":  "Reduce the number of conditional operators (6) used in the expression (maximum allowed 3).",
```

Comment:
what was this change? is it related to `not`?

Reply:
I think this is outdated. "Embber-MM" is no longer in "changed files".